### DISCUSSION TITLE
Management IP

### DISCUSSION THREAD

**Discussion by VincentKlanderman** [nautobot/nautobot#4135] [Ideas]
Rather than Having us create a device, then an interface, then assign an IP address to that interface. Have a management section where we can just put the management IP. I get the premise around having it assigned to an interface, but for devices like APs it would be nice to do just a quick assignment of a management IP address.

**Comment by lampwins** [comment]
Yeah this is reasonable. The relation between a device and the primary IP address is already direct in the data model, there is just additional validation logic that enforcing the IP existing on a device interface. I don't see a compelling reason to not make that scenario optional.
